wordpress-mobile/gutenberg-mobile
# Refactor Picker and fix state update on an unmounted component

Fixes #1049

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/15937

### To test:
- Try on WPAndroid and WPiOS
- Try every components that use the Picker: the block picker, the link editor, the image add and edit... and check that the modal shows and dismiss as expected and the value picked is the right one
